Add small note about rotating cluster after backup restore (#11733)
kubernetes/kops: docs/operations/etcd_backup_restore_encryption.md

After that, you can check again the endpoint and everything should be fixed.

After the restore is complete, api server should come back up, and you should have a working cluster.
Note that the api server might be very busy for a while as it changes the cluster back to the state of the backup. 

> You can consider temporarily increase the instance size of your masters and roll your worker nodes.

Reviewer: ```suggestion
You might consider temporarily increasing the instance size of your control plane.
```
